src/main/java/slimeknights/tconstruct/tables/tileentity/table/tinkerstation/ITinkerStationInventory.java

```diff
@@ -0,0 +1,14 @@
+package slimeknights.tconstruct.tables.tileentity.table.tinkerstation;
+
+import net.minecraft.item.ItemStack;
+import net.minecraft.util.NonNullList;
+import slimeknights.mantle.recipe.inventory.IReadOnlyInventory;
+
+public interface ITinkerStationInventory extends IReadOnlyInventory {
```

Comment:
Since we are using primarily `getAllInputStacks` and `getTinkerableStack`, I think deprecating and defaulting the vanilla methods is good. Notably `getStackInSlot`.